src/main.js

```diff
@@ -40,3 +39,12 @@ function randomizeBookCover() {
   )
   return bookCover;
 }
+
+function displayRandomButton() {
```

Comment:
This was a lot of fun. It is very few lines of code, but takes a lot of mental energy to understand. Great working on this with you Amanda!